examples/siro_sandbox/README_VR.md

```diff
@@ -0,0 +1,218 @@
+* Only `--app-state fetch` is supported.
+* Use the `examples/siro_sandbox/configs/fetch.yaml` configuration.
+
+#### Example
+
+```bash
+HABITAT_SIM_LOG=warning MAGNUM_LOG=warning \
+python examples/siro_sandbox/sandbox_app.py \
+--app-state fetch \
+--disable-inverse-kinematics \
+--never-end \
+--gui-controlled-agent-index 1 \
+--ang-speed "15" \
+--cfg examples/siro_sandbox/configs/fetch.yaml \
+--cfg-opts ~habitat.task.measurements.agent_blame_measure
+```
+
+## Unity Data Folder
+
+HSSD is not directly compatible with Unity. Furthermore, meshes must be simplified to run at an acceptable performance on the VR devices.
+
+Therefore, a script is provided so that you can process your HSSD scenes. Usage details are provided below.
+
+Note that the provided episodes support the following HSSD scenes:
+* 102344193
+* 102344280
+* 102817200
+* 103997424_171030444
+* 103997541_171030615
+
+### Dataset Processing
+
+#### Installation
+
+This step requires [Magnum](https://github.com/mosra/magnum) data processing tools. They are not built along with Habitat.
+
+It is recommended that you create a new `conda` environment so that it can be reused in the future without interfering with Habitat.
+
+1. Get the latest `magnum-tools` from [Magnum CI](https://github.com/mosra/magnum-ci/actions/workflows/magnum-tools.yml). Extract to a convenient location.
+2. Create a new `conda` environment:
+```
+conda create --name magnum python=3.10
+```
+3. Navigate to the `site-packages` of your new environment, e.g. `~/anaconda/envs/magnum/lib/python3.10/site-packages/`.
+4. Create a `magnum.pth` file in this directory.
+5. Add the absolute path to `magnum-tools`'s `python` folder to this file, e.g:
+```
+/home/USER/Documents/magnum-tools/linux-x64/python/
+```
+6. The Magnum libraries will now be included upon activating your `magnum` environment. Validate the installation as such:
+   * `conda activate magnum`
+   * `python -c "from magnum import math, meshtools, scenetools, trade"`
+
+#### Usage
+
+Example command:
+
+```
+python get_scene_object_glbs.py \
+--hssd-hab-root-dir path_to/hssd-hab/ \
+--hssd-models-root-dir path_to/hssd-models/ \
+--scenes 102344193 102344280 102817200 103997424_171030444 103997541_171030615
+```
+
+#### Importing Data Into Unity
+
+In Unity, use `Tools/Update Data Folder...`. From the dialog window, copy the path to the generated `data/hitl_simplified/data` in the `External Data Path` field. The resources will be imported into Unity.
+
+After this step, you may deploy the application to your headset. See [instructions](https://github.com/eundersander/siro_hitl_unity_client/blob/main/README.md).
+
+## Connection
+
+Upon launching the server, it will start listening for incoming connections. The client will attempt to connect to the addresses listed in `Android/data/com.meta.siro_hitl_vr_client/files/config.txt`. It rotates between the addresses periodically until a connection is established.
```

Comment:
This is a bit confusing. Are you suggesting here that the user run the client on the Quest headset at this point? I would suggest a more incremental approach as I gave in https://github.com/eundersander/siro_hitl_unity_client/blob/main/README.md (e.g. test in Unity editor before testing on headset; test with local file-based gfx-replay before trying client/server, etc.)